pkg/apis/pipeline/v1beta1/taskrun_types.go

```diff
@@ -304,8 +316,8 @@ type CloudEventDeliveryState struct {
 	// SentAt is the time at which the last attempt to send the event was made
 	// +optional
 	SentAt *metav1.Time `json:"sentAt,omitempty"`
-	// Error is the text of error (if any)
-	Error string `json:"message"`
+	// Message is the text of error (if any)
+	Message string `json:"message"`
```

Comment:
I believe this should also not be renamed as discussed in https://github.com/tektoncd/pipeline/pull/4402#discussion_r787932214

Reply:
Done.